price_service/server/package.json

```diff
@@ -17,9 +17,11 @@
   "license": "Apache-2.0",
```

Comment:
Can you bump the version of price service too? `npm version patch --no-git-tag-version`